csfieldguide/chapters/content/en/big-data/sections/the-three-vs.md

```diff
@@ -0,0 +1,53 @@
+
+One of the most extreme examples of data volume is YouTube uploads. As of October 2019, over [8 hours](https://everysecond.io/youtube) of video content is uploaded to YouTube every second!
+That's over 80 years worth of videos everyday, and YouTube has to be able to store all of this data to keep its services running.
+
+
+## Variety
+
+Variety refers to the many different types of data (plain text, images, videos, and many many more) that are generated and stored. Online transactions, emails, and even individual mouse clicks on a webpage are generating data of different types.
+
+{image file-path="img/chapters/different-data-types.png" alt="Icons representing seven different types of files or data points are shown. These are audio files, mouse clicks, online transactions, online messages, email, images, and a spreadsheet."}
+
+Big Data almost always uses more than one data type, and this contributes to the complexity of it.
+Different data types have different attributes and therefore the data doesn't simply fit into rows and columns of a spreadsheet or database (e.g. the dimensions of an image will not fit into the same field as the length of an audio clip).
+
+## Velocity
+
+There are two ways to measure Velocity: the speed at which the data is being generated, and the speed at which it is being {glossary-link term="data-processing"}processed{glossary-link end}.
+Big Data tends to be produced continually, and is available in real time.
+In many situations it also needs to be processed in real time, and processing this data as quickly as it is created is not easy!
+
+For example, every time you make a purchase using your EFTPOS card, that transaction is recorded and processed.
+In 2018 there were on average 50-60 EFTPOS transactions every single second [in New Zealand](https://www.paymentsnz.co.nz/resources/articles/nz-payments-stats-2018-in-review/), and this more than doubled in the Christmas period.
+When one of these transactions is made it doesn't just mean a message is sent to your bank and they take the money out of your account, there is much more than that happening.
+The EFTPOS machine must first establish a connection with a card processor network (such as Visa or MasterCard), who connect with your specific bank and pass on the transaction information to ask for approval.
+Checks are run to see if you have enough money, if your pin is correct, loyalty programs such as petrol points or airpoints might need to be computed, fraud checks might be run, and your bank may send data to a third party to perform some of these steps for them.
+Only after all this happens will your bank authorise (or decline it!) the transaction, send an approval back to the card processor network, then back to the EFTPOS machine saying the payment has been processed - and this all has to happen in a matter of seconds!
```

Comment:
take the 'it' out of the parentheses i.e. (or decline!)

Maybe use about a second rather than a number of seconds; I don't recall it taking that long

Use `&ndash;` instead of the hyphen